solutions/41.js

```diff
@@ -0,0 +1,21 @@
+const prime = (limit,a=2,b=[])=>{
+  if(a==limit){
+    return b;
+  }
+  const prime1 = (i=2)=>{
+    if(a%i==0){
+      if(a==i){
```

Comment:
this looks a little ugly here. This is better:
```
if( a !== i) return;
b.push(a);
```

seems like you are not returning any value. In that case, you don't even need to use return at all. The function automatically (by default) returns nothing

Reply:
thank you song, made the changes